### PR TITLE
added react pagination

### PR DESCRIPTION
**Pull request by silchyn**
[DEMO](https://silchyn.github.io/react_pagination/)

### CODE REVIEW

File: src/components/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -1 +1,87 @@
...
+          </a>
+        </li>
+      </ul>
+      <ul>
+        {visibleItems.map(item => (
+          <li data-cy="item" key={item}>
+            {`Item ${item}`}
+          </li>
+        ))}
+      </ul>
```
Comment (LordAsmodey) [inline src/components/Pagination/Pagination.tsx:84]: I think this is not related to the pagination component :)

**Review by LordAsmodey — APPROVED** [review]

**Review by ivan-kadykalo — APPROVED** [review]
🔥
